### ISSUE TITLE
"Grass" green on white - insufficient contrast

### ISSUE THREAD

**Issue by grugnog** [CivicActions/nebula#81]
This is the green on the "clear list" and "Link to this search" text. This one we can resolve with a change from  #00A159 to something like #008348  or #008839. Any thoughts from a design perspective?

**Comment by ghost** [comment]
"Link to this search" should the the same as the new body text color: 5B5D60. "clear list" a link color should be hex color: 006689.
